wiki/Beatmap_Editor/pt-br.md

```diff
@@ -0,0 +1,19 @@
+<img src="Compose.png" title="Editor do osu! em modo janela" alt="Editor do osu! em modo janela" width="200" />
+
+O **editor de beatmap** é a interface gráfica dentro do jogo do osu! para criar [beatmaps](beatmap "wikilink"). Ele é dividido em quatro modos:
+
+-   [Compose](Compose "wikilink"): Aqui é onde [Hit Objects](Hit_Object "wikilink") são colocados e editados. A maior parte do tempo gasto no editor é feito aqui.
+-   [Design](Design "wikilink"): Aqui é onde a Storyboard é feita, a maioria dos mapeadores só usarão o botão "Background Image/Video" aqui.
+-   [Timing](Timing "wikilink"): Aqui é onde o BPM e o deslocamento da música são inseridos. Esta janela também controla hitsounds, caso não sejam constantes em todo o mapa.
+-   [Song Setup](Song_Setup "wikilink"): Aqui é onde várias opções do mapa são definidas. Isso inclui cores de combo e configurações de dificuldade.
+
+Artigos relacionados
+----------------
+
+-   [Beatmapping](Beatmapping "wikilink") (Noções básicas de mapeamento)
+-   [Beat Snap Divisor](Beat_Snap_Divisor "wikilink")
+-   [Distance Snap](Distance_Snap "wikilink")
+-   [Custom Sample Overrides](Custom_Sample_Overrides "wikilink")
+-   [Kiai Time](Kiai_Time "wikilink")
```

Comment:
All these links are outdated. I'll link them below how you should add them, respectively, after each [section].

- (/wiki/Beatmaps)
- (/wiki/Beatmap_Editor/Compose)
- (/wiki/Hit_Objects)
- (/wiki/Beatmap_Editor/Design)
- (/wiki/Storyboards)
- (/wiki/Beatmap_Editor/Timing)
- (/wiki/Beatmap_Editor/Song_Setup)
- (/wiki/Beatmap_Editor/Beat_Snap_Divisor)
- (/wiki/Beatmap_Editor_Guides/Custom_Sample_Overrides)
- (/wiki/Beatmap_Editor/Distance_Snap)
- (/wiki/Beatmap_Editor/Kiai_Time)